src/main/java/io/kestra/plugin/aws/AbstractConnection.java

```diff
@@ -26,6 +26,8 @@ public abstract class AbstractConnection extends Task implements AbstractConnect
 
     protected String endpointOverride;
 
+    private Boolean useDefaultAsyncClient;
```

Comment:
As it's only supported for S3, better to move it to AbstractS3 instead.

Reply:
I think it will be better named as `compability` mode or something like this as users may not be aware of when they need to use this client.
Then on the description of the property explain that this property will use the AsynS3Client instead of the S3CrtAsynClient which maximize compatibility with S3-compatible services but restrict uploads and downloads to 2GB.